src/builtins/coefficient/index.ts

```diff
@@ -3,7 +3,7 @@ import {z} from 'zod';
 import {ExecutePlugin, PluginParams} from '../../types/interface';
 
 import {validate} from '../../util/validations';
-import {ERRORS} from '../../util/errors';
+import {ERRORS} from '@grnsft/if-core';
```

Comment:
Move this to the top and update it in the other files

Reply:
it's coming in next PR